docker-app/qfieldcloud/core/management/commands/dequeue.py

```diff
@@ -15,6 +16,13 @@
     ProcessProjectfileJobRun,
 )
 
+if os.environ.get("DEBUG_BUILD"):
+    import debugpy
+
+    debugpy.listen(("0.0.0.0", 5679))
+    print("debugpy ready for debugger on port 5679... 🐛")
```

Comment:
I'd suggest not starting debugpy by default (could happen that this lands on prod for some reason, in which case it's a huge security issue, and there's probably some overhead ?)

We could instead documenter debugging the workers as above: 
```
docker-compose run worker_wrapper python -m debugpy --listen 0.0.0.0:5678 manage.py dequeue
```

This way it's also more symetrical between app and worker_wrapper.

Reply:
Ok, I see what you mean. I have removed it, now trying to provide docs how should it work....